scalac-scoverage-plugin/src/test/scala/scoverage/ScoverageCompiler.scala

```diff
@@ -17,7 +19,7 @@ object ScoverageCompiler {
     case _                                       => ScalaVersion
   }
 
-  def classPath = getScalaJars.map(_.getAbsolutePath) :+ sbtCompileDir.getAbsolutePath :+ runtimeClasses.getAbsolutePath
+  def classPath = (getScalaJars ++ getScalaJsJars ++ List(sbtCompileDir, runtimeClasses)).map(_.getAbsolutePath)
```

Comment:
You should not put `scalajs-library.jar` on the classpath of the compiler, ever. It should be passed to the `-classpath` setting of the compiler so that it ends up on the classpath of the project being compiled instead.

Reply:
Addressed in https://github.com/scoverage/scalac-scoverage-plugin/pull/281/commits/4dffb6c764ef5e950c5cfd23be4a7fd7f95f9781
I am uncertain about this, but `addToClassPath` seems to append the file at the tend of classpath.